channels/slack/src/main/kotlin/com/justai/jaicf/channel/slack/SlackChannel.kt

```diff
@@ -38,10 +40,21 @@ class SlackChannel private constructor(
     private constructor(botApi: BotApi, urlPrefix: String): this(botApi) {
         val config = SlackConfig().apply {
             methodsEndpointUrlPrefix = urlPrefix
+            methodsConfig = MethodsConfig().apply {
+                isStatsEnabled = false
+            }
         }
 
         val slack = Slack.getInstance(config)
-        app = App(AppConfig.builder().slack(slack).build())
+        app = App(AppConfig.builder()
```

Comment:
Should we do this configuration by ourselves? Is there a way to let a user configure the channel?

Reply:
This constructor is invoked from JAICP channel factory, and it is configured to process messages without OAuth, signature and etc. This is our internal logic concerning integration with JAICP, so it should not be configured externally, as I think.

Reply:
Okay, then maybe we should give a user a way to configure the channel inside another constructor? 
`com.justai.jaicf.channel.slack.SlackChannelConfig` seems like a very weak configurability

Reply:
I agree. Middleware was added to SlackChannelConfig.